go/kbfs/libkbfs/favorites.go

```diff
@@ -661,6 +661,10 @@ func (f *Favorites) handleReq(req *favReq) (err error) {
 }
 
 func (f *Favorites) loop() {
+	if f.bufferedInterval <= 0 {
```

Comment:
I think it's slightly cleaner to use `if f.disabled` here, but either way is fine.